RELEASE-NOTES.txt

```diff
@@ -7,6 +7,7 @@
 6.5
 -----
 - [*] Fix: Product images with non-latin characters in filenames now will load correctly and won't break Media Library. [https://github.com/woocommerce/woocommerce-ios/pull/3935]
+- [*] Fix: The screen to select images from the Media Library would sometimes crash when the library had a specific number of images. [https://github.com/woocommerce/woocommerce-ios/pull/4070]
```

Comment:
Note: The Release Notes won't get picked up when doing a PR targeting `release/*`, because those release notes have already been sent to Editorial middle of last week, then to our translation vendor last Sunday.

I'm ok keeping that change still (so we have the history of what was shipped in 6.5 for ourselves internally), just wanted to point it out to be sure you knew.

Reply:
I was wondering about that, and what will happen when the 6.5 branch gets merged into develop. The idea was to have that item in 6.5 just to have the history of the changes, but now it might appear in both after merging? 😬

Reply:
Yeah it will appear in both indeed 🤔  🤷  🙃